python/py-pgspecial/Portfile

```diff
@@ -30,6 +31,4 @@ if {${name} ne ${subport}} {
                         port:py${python.version}-sqlparse
```

Comment:
I would be surprised if these are all only build dependencies... py-click and py-sqlparse are not needed at runtime? Also, it seems to be missing a dependency when looking at upstream's [setup.py](https://github.com/dbcli/pgspecial/blob/0d445739937015de06dd4d9a959b12a3aa3be2ef/setup.py#L24).